hologram_dashboard/fetcher/enums.proto

```diff
@@ -0,0 +1,53 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+syntax = "proto2";
+package hologramAvailability;
+
+enum DataType {
+    PLAY_COUNTRY = 0;
```

Comment:
In proto enum, 0 is used for UNKNOWN_DATA_TYPE/UNSPECIFED_DATA_TYPE/DEFAULT_DATA_TYPE.

Valid enum values should be >0.